policy/modules/system/systemd.te

```diff
@@ -1126,12 +1126,13 @@ optional_policy(`
 # Update Done local policy
 #
 
-allow systemd_update_done_t systemd_update_run_t:file manage_file_perms;
+allow systemd_update_done_t self:process setfscreate;
 
-dev_write_kmsg(systemd_update_done_t)
+allow systemd_update_done_t systemd_update_run_t:file manage_file_perms;
 
 files_etc_filetrans(systemd_update_done_t, systemd_update_run_t, file, ".updated")
 files_var_filetrans(systemd_update_done_t, systemd_update_run_t, file, ".updated")
```

Comment:
It would seem that these two filetranses are not necessary, if it is looking up the file contexts.

Reply:
They may have been needed with older versions of systemd, which is why I have not removed them.

Reply:
Fair enough.